OnionBrowser/Browsing/Tab.swift

```diff
@@ -176,7 +176,19 @@ class Tab: UIView {
 		return refresher
 	}()
 
+	private var _snapshot: UIImage? = nil
 
+	var snapshot: UIImage? {
+			if _snapshot == nil {
+				_snapshot = layer.makeSnapshot(scale: 1.0)
+			}
+			return _snapshot!
```

Comment:
This is a way to produce an unnecessary crash. Your `makeSnapshot()` can return `nil`.

Reply:
fixed